contracts/foundry.toml

```diff
@@ -2,5 +2,6 @@
 src = "src"
 out = "out"
 libs = ["lib"]
+evm_version = "paris"
```

Comment:
:shrug: we don't have this anywhere else. Intentionally?
https://github.com/search?q=org%3Aoasisprotocol+path%3Afoundry.toml&type=code